src/sentry/models/commitfilechange.py

```diff
@@ -0,0 +1,29 @@
+from __future__ import absolute_import, print_function
+
+from django.db import models
+
+from sentry.db.models import (
+    BoundedPositiveIntegerField, FlexibleForeignKey, Model, sane_repr
+)
+
+
+class CommitFileChange(Model):
```

Comment:
We should add this explicitly to `delete_organization` before `Commit`. I understand that it'll get yanked by `Commit` because of foreign key, but `CommitFileChange` is going to potentially have massively more rows tied to each `Commit`, so to avoid yanking tons more rows than intended at once time, let's incrementally delete all of these first.